spyder/plugins/plots/widgets/tests/test_plots_widgets.py

```diff
@@ -235,6 +235,28 @@ def test_go_prev_next_thumbnail(figbrowser, tmpdir, fmt):
     assert figbrowser.figviewer.figcanvas.fig == figs[1]
 
 
+def test_scroll_to_item(figbrowser, tmpdir, qtbot):
+    """Test scroll to the item of ThumbnailScrollBar."""
+    nfig = 10
+    add_figures_to_browser(figbrowser, nfig, tmpdir, 'image/png')
+    figbrowser.setFixedSize(500, 500)
+
+    for __ in range(nfig // 2):
+        figbrowser.go_next_thumbnail()
+        qtbot.wait(500)
+
+    scene = figbrowser.thumbnails_sb.scene
+
+    spacing = scene.verticalSpacing()
+    height = scene.itemAt(0).sizeHint().height()
+    height_view = figbrowser.thumbnails_sb.scrollarea.viewport().height()
+
+    expected = (spacing * 5) + (height * 4) - ((height_view - height) // 2)
```

Comment:
Would it be possible to replace the hard-coded 5 and 4 values by `nfig // 2` and `nfig//2 - 1)` ?

I think this would make the test easier to understand.

Reply:
Indeed, I was not sure what those numbers meant